carbon-mb/components/transports/mqtt/src/main/java/org/wso2/carbon/andes/transports/mqtt/broker/BrokerVersion.java

```diff
@@ -43,7 +43,7 @@
     /**
      * We maintain the instance of the broker which corresponds to its version
      */
-    private Broker broker;
+    private transient Broker broker;
```

Comment:
any reason as to why this is marked transient explicitly? are we serializing instances of this class?